### PR TITLE
Feature/amp

### PR DESCRIPTION
**Pull request by domjngo**
On test https://test.media.nationalarchives.gov.uk/

### CODE REVIEW

**Comment by domjngo** [comment]
@punalsc here are the files to focus on:

- [x] partials/player
- [x] partials/credits
- [ ] partials/gallery - Unable to test this partial, as it does not exist on the test site.
- [ ] partials/slides - On the front-end the javascript does not seem to be working - internal error 500. Fonts used are not loading, which is causing errors in the console.
- [x] partials/transcription
- [x] content-series
- [x] content-amp-catagories
- [x] page-amp-home

Thanks!

**Comment by punalsc** [comment]
Thumbnail on landing page should be positioned above the description on landing page, in mobile view.

**Comment by punalsc** [comment]
`partials/slides.php` and `partials/transcription.php` seems to be having the same codebase, which goes against the DRY principles. For now this is fine, but i would consider refactoring the code to combine them as one function.

File: header-blog.php [hunk 1]
```diff
@@ -24,12 +28,25 @@
...
+									<div class="col-xs-6 col-sm-12 col-md-6">
+										<a href="<?php echo esc_url( get_category_link( $video_id ) ); ?>" class="button button-icon icon-video">
+											Video
+										</a>
+										<a href="<?php echo esc_url( get_category_link( $audio_id ) ); ?>" class="button button-icon icon-audio">
+											Audio
+										</a>
+									</div>
+								<?php } else { ?>
 								<form action="" method="get" class="month-list col-xs-6 col-sm-12 col-md-6">
```
Comment (punalsc) [inline header-blog.php]: Please check if the form needs a aria attribute.

**Review by punalsc — APPROVED** [review]

**Pull request merged**